Description: Drop documentation about removed pkgimport.conf.
Origin: commit, revision id: jelmer@jelmer.uk-20190217034531-vw7oc2bo5hdd41jn
Author: Jelmer VernooÄ³ <jelmer@jelmer.uk>
Last-Update: 2019-02-17
Applied-Upstream: no
X-Bzr-Revision-Id: jelmer@jelmer.uk-20190217034531-vw7oc2bo5hdd41jn

=== modified file 'doc/developers/configuration.txt'
--- old/doc/developers/configuration.txt	2017-07-30 16:59:50 +0000
+++ new/doc/developers/configuration.txt	2019-02-17 03:45:31 +0000
@@ -154,20 +154,6 @@
 though). The ``bzr-bookmarks`` plugin defines a ``BOOKMARKS`` section there
 for example.
 
-pkgimport.conf
-==============
-
-The Ubuntu package importer defines a store and two stacks involving
-``pkgimport.conf``. A no-name section contains the options common to all
-packages and sections named after their corresponding package can also be
-defined.
-
-The ``ImporterStack`` uses ``locations.conf`` and the no-name section in
-``pkgimport.conf`` for the importer options.
-
-The ``PackageStack`` uses only ``pkgimport.conf`` and uses the section name
-after the package followed by the no-name section.
-
 location.conf
 =============
 

